### PR TITLE
Add rubocop, rubocop-rspec

### PR DESCRIPTION
**Pull request by andyw8**

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+# Offense count: 2
+# Configuration parameters: Max.
+RSpec/ExampleLength:
+  Exclude:
+    - 'spec/transaction_spec.rb'
+
+# Offense count: 5
+# Configuration parameters: CustomTransform, IgnoreMethods.
+RSpec/FilePath:
```
Comment (andyw8) [inline .rubocop_todo.yml]: I opened https://github.com/financeit/marqeta/pull/6 to resolve this.

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+# `rubocop --auto-gen-config`
+# on 2018-03-03 10:25:38 -0500 using RuboCop version 0.52.1.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 11
+# Configuration parameters: CountComments, ExcludedMethods.
+Metrics/BlockLength:
```
Comment (andyw8) [inline .rubocop_todo.yml]: Some of this config will need tweaking, but I think it's good to have a starting point.

File: .rubocop_todo.yml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+# Offense count: 7
+# Configuration parameters: AggregateFailuresByDefault.
+RSpec/MultipleExpectations:
+  Max: 6
+
+# Offense count: 3
+RSpec/SubjectStub:
+  Exclude:
+    - 'spec/api_object_spec.rb'
+    - 'spec/card_spec.rb'
```
Comment (andyw8) [inline .rubocop_todo.yml]: The entries for `card_spec` and `transaction_spec` should be resolved by https://github.com/financeit/marqeta/pull/8.

**Comment by dubroe** [comment]
@andyw8 we're having a failed build due to an incompatible ruby version

File: .rubocop_todo.yml (hunk 1 above)
Comment (dubroe) [reply .rubocop_todo.yml]: So @andyw8 should we remove these exclusions since we'll be moving the files over?

Comment (andyw8) [reply .rubocop_todo.yml]: Let's merge this first, then I'll update #6.

**Comment by andyw8** [comment]
@dubroe looks like it was defaulting to Ruby 2.1.7. I changed the config to use 2.2.3 and it passed.

**Pull request merged**